docs/make.jl

```diff
@@ -52,58 +45,83 @@ function generate_example_page(sections)
             ```
             $(cmd_jl_str)
             ```
-            The output will be like this:
-            ```@raw html
-            <div id="$(name)"></div>
-            <script>
-                AsciinemaPlayer.create('assets/$(name).cast', document.getElementById('$(name)'), {
-                    cols: 80,
-                    rows: 24
-                });
-            </script>
-            ```
             """
+
+            if mode == :asciinema
```

Comment:
Are there any plans to add modes other than `:disabled` and `:asciinema` for the future?

Reply:
I was thinking that `:video` can be added in the future to record the results with Sixel.jl. (x-ref: https://github.com/AtelierArith/Replay.jl/pull/54#discussion_r1161409540)

Reply:
I see 🫡